installation/language/de-DE/joomla.ini

```
@@ -77,28 +77,28 @@ INSTL_DATABASE_NAME_LABEL="Datenbankname"
 INSTL_DATABASE_NAME_MSG_POSTGRES="Der Datenbankname ist ungültig. Der Datenbankname muss mit einem Buchstaben beginnen, gefolgt von alphanumerischen Zeichen."
 INSTL_DATABASE_NAME_MSG_MYSQL="Der Datenbankname ist ungültig. Der Datenbankname darf die folgenden Zeichen nicht enthalten: \ / ."
 INSTL_DATABASE_NO_SCHEMA="Für diesen Datenbanktyp existiert kein Datenbankschema."
-INSTL_DATABASE_PASSWORD_DESC="Entweder ein selbsterstelltes Passwort oder das vom Webhoster bereitgestellte Passwort verwenden."
-INSTL_DATABASE_PREFIX_DESC="Geben Sie ein Tabellenpräfix ein oder verwenden Sie das zufällig generierte."
+INSTL_DATABASE_PASSWORD_DESC="Datenbankpasswort (selbst erstellt oder vom Webhoster vorgegeben)."
+INSTL_DATABASE_PREFIX_DESC="Tabellenpräfix (wie generiert oder selbst vergeben)."
 INSTL_DATABASE_PREFIX_MSG="Der Tabellenpräfix muss mit einem Buchstaben beginnen. Danach können optional alphanumerische Zeichen folgen. Er muss mit einem Unterstrich enden."
 INSTL_DATABASE_RESPONSE_ERROR="Der Installationsvorgang ist fehlgeschlagen."
 INSTL_DATABASE_TYPE_DESC="Datenbanktyp auswählen."
-INSTL_DATABASE_USER_DESC="Entweder ein erstellter Benutzername oder ein vom Webhoster zur Verfügung gestellter Benutzername."
+INSTL_DATABASE_USER_DESC="Datenbank-Benutzername (selbst gewählt oder vom Webhoster vorgegeben)."
```

```suggestion
INSTL_DATABASE_USER_DESC="Datenbank-Benutzername (selbst gewählt oder vom Webhoster vorgegeben)."
```
Oder wir nehmen einfach erstmal das?

---

Wir bleiben jetzt erstmal dabei. Mit gewählt passt das hier schon. Danke @chmst